test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/Options/PackageSourceMappingPageTests.cs

```diff
@@ -306,24 +306,25 @@ public async Task SetValueAsync_PreviousMappingsToNonexistantSources_AddNewMappi
             string addNewPackageSourceName4 = "unitTestingSourceName4";
             string packageIdPattern = "Contoso.*";
 
-            var unitTestingSourceMapping1 = new PackageSourceMappingSourceItem("unitTestingSourceName1", [new PackagePatternItem(packageIdPattern)]);
-            var nonExistantSourceInSourceMapping2 = new PackageSourceMappingSourceItem("nonExistantSourceName2", [new PackagePatternItem(packageIdPattern)]);
-            var unitTestingSourceMapping3 = new PackageSourceMappingSourceItem("unitTestingSourceName3", [new PackagePatternItem(packageIdPattern)]);
-
-            _vsSettings.AddOrUpdate(ConfigurationConstants.PackageSourceMapping, unitTestingSourceMapping1);
-            _vsSettings.AddOrUpdate(ConfigurationConstants.PackageSourceMapping, nonExistantSourceInSourceMapping2);
-            _vsSettings.AddOrUpdate(ConfigurationConstants.PackageSourceMapping, unitTestingSourceMapping3);
-
-            PackageSourceMappingPage instance = CreateInstance(_vsSettings);
-
             string sourceName1 = "unitTestingSourceName1";
             string sourceUrl1 = "https://testsource1.com";
 
-            // nonExistantSourceName2 is not added here intentionally.
+            string nonExistantSourceName2 = "nonExistantSourceName2";
```

Comment:
Nit: minor misspelling

```suggestion
            string nonExistentSourceName2 = "nonExistentSourceName2";
```

Reply:
Great catch, I'll refactor that next time